Prevent implicit config file accesses by OpenSSL. libretls does the same.

--- acctproc.c
+++ acctproc.c
@@ -475,6 +475,7 @@ acctproc(int netsock, const char *acctkey, enum keytype keytype)
 
 	/* File-system, user, and sandbox jailing. */
 
+	OPENSSL_init_crypto(OPENSSL_INIT_NO_LOAD_CONFIG, NULL);
 	ERR_load_crypto_strings();
 
 	if (pledge("stdio", NULL) == -1) {
--- keyproc.c
+++ keyproc.c
@@ -110,6 +110,7 @@ keyproc(int netsock, const char *keyfile, const char **alts, size_t altsz,
 
 	/* File-system, user, and sandbox jail. */
 
+	OPENSSL_init_crypto(OPENSSL_INIT_NO_LOAD_CONFIG, NULL);
 	ERR_load_crypto_strings();
 
 	if (pledge("stdio", NULL) == -1) {
--- revokeproc.c
+++ revokeproc.c
@@ -87,6 +87,7 @@ revokeproc(int fd, const char *certfile, int force,
 
 	/* File-system and sandbox jailing. */
 
+	OPENSSL_init_crypto(OPENSSL_INIT_NO_LOAD_CONFIG, NULL);
 	ERR_load_crypto_strings();
 
 	if (pledge("stdio", NULL) == -1) {
